class AlreadyExisted(Exception):
    pass


class NotFound(Exception):
    pass
